datasets/austin_crime/crime/pipeline.yaml

```diff
@@ -0,0 +1,193 @@
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "KubernetesPodOperator"
+
+      # Task description
+      description: "Run CSV transform within kubernetes pod"
+
+      args:
+
+        task_id: "austin_crime_transform_csv"
+
+        # The name of the pod in which the task will run. This will be used (plus a random suffix) to generate a pod id
+        name: "crime"
+
+        # The namespace to run within Kubernetes. Always set its value to "default" because we follow the guideline that KubernetesPodOperator will only be used for very light workloads, i.e. use the Cloud Composer environment's resources without starving other pipelines.
+        namespace: "default"
+
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+                - matchExpressions:
+                    - key: cloud.google.com/gke-nodepool
+                      operator: In
+                      values:
+                        - "pool-e2-standard-4"
+
+
+        image_pull_policy: "Always"
+
+        # Docker images will be built and pushed to GCR by default whenever the `scripts/generate_dag.py` is run. To skip building and pushing images, use the optional `--skip-builds` flag.
+        image: "{{ var.json.austin_crime.container_registry.run_csv_transform_kub }}"
+
+        # Set the environment variables you need initialized in the container. Use these as input variables for the script your container is expected to perform.
+        env_vars:
+          SOURCE_URL: >-
+            ["gs://pdp-feeds-staging/Austin_Crime/Annual_Crime_2014.csv","gs://pdp-feeds-staging/Austin_Crime/Annual_Crime_Dataset_2015.csv","gs://pdp-feeds-staging/Austin_Crime/2016_Annual_Crime_Data.csv"]
+          SOURCE_FILE: >-
+            ["files/data1.csv","files/data2.csv","files/data3.csv"]
+          TARGET_FILE: "files/data_output.csv"
+          TARGET_GCS_BUCKET: "{{ var.json.shared.composer_bucket }}"
+          TARGET_GCS_PATH: "data/austin_crime/crime/data_output.csv"
+          FILE_PATH: "files/"
+          CSV_HEADERS: >-
+            ["unique_key","address","census_tract","clearance_date","clearance_status","council_district_code","description","district","latitude","longitude","location","location_description","primary_type","timestamp","x_coordinate","y_coordinate","year","zipcode"]
+          RENAME_MAPPINGS: >-
+            {"GO Primary Key" : "unique_key","Council District" : "council_district_code","GO Highest Offense Desc" : "description","Highest NIBRS/UCR Offense Description" : "primary_type","GO Report Date" : "timestamp","GO Location" : "location_description","Clearance Status" : "clearance_status","Clearance Date" : "clearance_date","GO District" : "district","GO Location Zip" : "zipcode","GO Census Tract" : "census_tract","GO X Coordinate" : "x_coordinate","GO Y Coordinate" : "y_coordinate","Location_1" : "address1"}
```

Comment:
^

Reply:
same as above

Reply:
Actually, what you can do here is parse the multi-line string
```jsonc
{
  "GO Primary Key" : "unique_key",
  "Council District" : "council_district_code",
  // and so on
}
```
using the `json` library in your transform script:
```python
import json
rename_mappings = json.loads(input_string)
```
Here's an example, showing that it indeed includes `\n` in the string, but it can still be mapped to a Python dict:

<img width="568" alt="Screen Shot 2021-09-28 at 4 12 35 PM" src="https://user-images.githubusercontent.com/1208372/135158922-2a9b82d2-4d62-423d-90f3-5f78b8d16326.png">

Reply:
As discussed , as now we are going with this long line of list . Hence, not making changes.